src/Microsoft.VisualStudio.LanguageServices.Razor/VisualStudioRazorParser.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
+using Microsoft.VisualStudio.Text;
+using ITextBuffer = Microsoft.VisualStudio.Text.ITextBuffer;
```

Comment:
Is this needed? Does the Razor type still exist?

What if we got rid of `Microsoft.AspNetCore.Razor.Language.Legacy;` and just qualified those things?